radlab-ui/webapp/src/components/forms/fields/MapField.tsx

```diff
@@ -62,6 +62,13 @@ const MapField: React.FC<MapField> = ({ variable, validate }) => {
         {(FieldArrayProps) => {
           const { form } = FieldArrayProps
           const { values } = form
+          if (
```

Comment:
`billing_budget_labels` themselves have default value of `{}` in [terraform variables.tf](https://github.com/GoogleCloudPlatform/rad-lab/blob/66a1ea10f4334d8637b7288b4c5bd3f7f50da13d/modules/billing_budget/variables.tf#L62) so that is expected.

There also another variable called [`resource_creator_identity`](https://github.com/GoogleCloudPlatform/rad-lab/blob/66a1ea10f4334d8637b7288b4c5bd3f7f50da13d/modules/billing_budget/variables.tf#L175) which has default value of `""` I don't see that in above example.